Cargo.toml

```diff
@@ -19,17 +19,35 @@ debug-assertions = true
 opt-level = "z"
 overflow-checks = true # DEV: Do not remove this check - doing so will create vulnerabilities
 debug = 0
-strip = "symbols"
+strip = "none"
 debug-assertions = false
 panic = "abort"
 codegen-units = 1
 lto = true
 
+[workspace.dependencies.certora]
+git = "https://github.com/Certora/solana-cvt.git"
+branch = "dev-soroban"
+
+[workspace.dependencies.certora-soroban]
+git = "https://github.com/Certora/solana-cvt.git"
+branch = "abakst/dev-soroban"
```

Comment:
these will change after it is merged right? We need to mention the repo/branch in the report, so i ask. Might be good if it is all in a single branch if possible.

Reply:
i mentioned the dev-soroban branch to the report fwiw